src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--4 {
+    .stars__star:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
```

Comment:
It's a good practice to create a mixin for repetitive code like the one that changes the star images, as it will make the code more maintainable and scalable when dealing with different star ratings (e.g., 1 to 5 stars).